subprojects/core/src/integTest/groovy/org/gradle/api/ConfigurationOnDemandIntegrationTest.groovy

```diff
@@ -378,6 +390,7 @@ project(':api') {
         fixture.assertProjectsConfigured(":", ":b", ":a")
     }
 
+    @ToBeFixedForIsolatedProjects(because = "Configure projects from root, buildDependents is not IP compatible")
```

Comment:
💅 Probably better to always keep the API name closer to the `because = "` for easier search